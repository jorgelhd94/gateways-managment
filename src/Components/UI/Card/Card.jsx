import React from 'react';
import PropTypes from 'prop-types';

const Card = ({ children }) => {
  return (
    <div className="flex flex-col m-4 px-4 py-8 bg-white rounded-lg shadow dark:bg-gray-800 sm:px-6 md:px-8 lg:px-10">
      {children}
    </div>
  );
};

Card.propTypes = {
  children: PropTypes.node
};

export default Card;
