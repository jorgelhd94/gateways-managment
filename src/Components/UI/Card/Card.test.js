import React from 'react';
import renderer from 'react-test-renderer';
import Card from './Card';

test('Card render correctly', () => {
  const component = renderer.create(<Card>Hello</Card>);
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
