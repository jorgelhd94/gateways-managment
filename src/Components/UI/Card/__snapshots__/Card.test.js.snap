// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card render correctly 1`] = `
<div
  className="flex flex-col m-4 px-4 py-8 bg-white rounded-lg shadow dark:bg-gray-800 sm:px-6 md:px-8 lg:px-10"
>
  Hola
</div>
`;
